src/main/resources/com/puppycrawl/tools/checkstyle/checks/metrics/messages_fi.properties

```diff
@@ -5,4 +5,5 @@ cyclomaticComplexity=Syklomaattinen monimutkaisuus on {0,number,integer} (max sa
 ncss.class=Sävyä NCS tämän luokan on {0, number,integer} (max sallittu on {1, number,integer}).
 ncss.file=Sävyä NCS Tämän tiedoston on {0, number,integer} (max sallittu on {1, number,integer}).
 ncss.method=Sävyä NCS Tämän menetelmän on {0, number,integer} (max sallittu on {1, number,integer}).
+ncss.record=Sävyä NCSS tämän ennätys on {0, number,integer} (max sallittu on {1, number,integer}).
```

Comment:
I have no idea why all other messages in Finnish have "NCS" instead of "NCSS".
Looks like a bug. I played with the google/microsoft/yandex translators a bit and it looks like there should be "NCSS" here after all.